grafana-plugin/src/components/SchedulesFilters/SchedulesFilters.tsx

```diff
@@ -1,59 +1,94 @@
-import React, { useCallback, useMemo } from 'react';
+import React, { ChangeEvent, useCallback } from 'react';
 
-import { DatePickerWithInput, Field, HorizontalGroup, RadioButtonGroup } from '@grafana/ui';
+import { Field, Icon, Input, RadioButtonGroup } from '@grafana/ui';
 import cn from 'classnames/bind';
-import moment from 'moment-timezone';
 
-import { dateStringToOption, optionToDateString } from './SchedulesFilters.helpers';
-import { SchedulesFiltersType } from './SchedulesFilters.types';
+import { ScheduleType } from 'models/schedule/schedule.types';
 
-import styles from './SchedulesFilters.module.css';
+import styles from './SchedulesFilters.module.scss';
+import { SchedulesFiltersType } from './SchedulesFilters.types';
 
 const cx = cn.bind(styles);
 
 interface SchedulesFiltersProps {
   value: SchedulesFiltersType;
   onChange: (filters: SchedulesFiltersType) => void;
-  className?: string;
 }
 
-const SchedulesFilters = ({ value, onChange, className }: SchedulesFiltersProps) => {
-  const handleDateChange = useCallback(
-    (date: Date) => {
-      onChange({ selectedDate: moment(date).format('YYYY-MM-DD') });
+const SchedulesFilters = (props: SchedulesFiltersProps) => {
```

Comment:
Hmm, I haven't touched these parts. Don't know why these changes show up in here